### PR TITLE
Adding more replies to 8ball and rate answers 

### PR DESCRIPTION
**Pull request by guipleite**
Adding more replies to 8ball and rate answers for issue #117

### CODE REVIEW

File: CompatBot/Commands/Misc.cs [hunk 1]
```diff
@@ -18,29 +18,32 @@ internal sealed class Misc: BaseCommandModuleCustom
...
+            // 43
             "Ya fo sho", "Fo shizzle mah nizzle", "Yuuuup", "Da", "Affirmative", // 5
             "Sure", "Yeah, why not", "Most likely", "Sim", "Oui",
             "Heck yeah!", "Roger that", "Aye!", "Yes without a doubt m8!", "<:cell_ok_hand:324618647857397760>",
             "Don't be an idiot. YES.", "Mhm!", "Many Yes", "Yiss", "Sir, yes, Sir!", 
             "Yah!", "Ja", "Umu!", "Make it so", "Sure thing",
             "Certainly", "Of course", "Definitely", "Indeed", "Much yes",
             "Consider it done", "Totally", "You bet", "Yup", "Yep",
-            "Certainly", "Yarp",
+            "Certainly", "Yarp", "Very well", "Will do", "Aye mate",
```
Comment (13xforever) [inline CompatBot/Commands/Misc.cs]: “Certainly” is a duplicate, “very well” and “will do” do not work very well as an answer (“should I wash the dishes?” “Will do” “um what”)

File: CompatBot/Commands/Misc.cs [hunk 2]
```diff
@@ -18,29 +18,32 @@ internal sealed class Misc: BaseCommandModuleCustom
...
+            "Right on it", "Got it down to a science", "Without fail",
 
-            // 20
+            // 24
             "Maybe", "I don't know", "I don't care", "Who cares", "Maybe yes, maybe not",
             "Maybe not, maybe yes", "Ugh", "Probably", "Ask again later", "Error 404: answer not found",
             "Don't ask me that again", "You should think twice before asking", "You what now?", "Ask Neko", "Ask Ani",
             "Bloody hell, answering that ain't so easy", "I'm pretty sure that's illegal!", "What do *you* think?", "Only on Wednesdays", "Look in the mirror, you know the answer already",
+            "Don't know, don't care", "*visible confusion*", "Have you googled it?", "Oh, I don't think so.",
```
Comment (13xforever) [inline CompatBot/Commands/Misc.cs]: Last one is a negative response, no?

File: CompatBot/Commands/Misc.cs [hunk 3]
```diff
@@ -18,29 +18,32 @@ internal sealed class Misc: BaseCommandModuleCustom
...
             "Ya fo sho", "Fo shizzle mah nizzle", "Yuuuup", "Da", "Affirmative", // 5
             "Sure", "Yeah, why not", "Most likely", "Sim", "Oui",
             "Heck yeah!", "Roger that", "Aye!", "Yes without a doubt m8!", "<:cell_ok_hand:324618647857397760>",
             "Don't be an idiot. YES.", "Mhm!", "Many Yes", "Yiss", "Sir, yes, Sir!", 
             "Yah!", "Ja", "Umu!", "Make it so", "Sure thing",
             "Certainly", "Of course", "Definitely", "Indeed", "Much yes",
             "Consider it done", "Totally", "You bet", "Yup", "Yep",
-            "Certainly", "Yarp",
+            "Certainly", "Yarp", "Very well", "Will do", "Aye mate",
+            "Right on it", "Got it down to a science", "Without fail",
```
Comment (13xforever) [inline CompatBot/Commands/Misc.cs]: Similar issues of incoherent conversation flow with the science reply, and maybe with “right on it”

File: CompatBot/Commands/Misc.cs [hunk 4]
```diff
@@ -18,29 +18,32 @@ internal sealed class Misc: BaseCommandModuleCustom
...
-            // 29
+            // 34
             "Nah mate", "Nope", "Njet", "Of course not", "Seriously no",
             "Noooooooooo", "Most likely not", "Não", "Non", "Hell no",
             "Absolutely not", "Nuh-uh!", "Nyet!", "Negatory!", "Heck no",
             "Nein!", "I think not", "I'm afraid not", "Nay", "Yesn't",
             "No way", "Certainly not", "I must say no", "Nah", "Negative",
-            "Definitely not", "No way, Jose", "Not today", "Narp",
+            "Definitely not", "No way, Jose", "Not today", "Narp", "Not in a million years", 
+            "I’m afraid I can’t let you do that Dave.", "This mission is too important for me to allow you to jeopardize it.", "I shall not", "By *no* means",
```
Comment (13xforever) [inline CompatBot/Commands/Misc.cs]: Again, “I shall not” is not really an answer that would work

File: CompatBot/Commands/Misc.cs [hunk 5]
```diff
@@ -63,15 +66,16 @@ internal sealed class Misc: BaseCommandModuleCustom
             "So cute!", "It's so nice, I think about it every day!", "😊 Never expected to be this pretty!", "It's overflowing with charm!", "Filled with passion!",
             "A love magnet", "Pretty Fancy", "Admirable", "Sweet as a candy", "Delightful",
             "Enchanting as the Sunset", "A beacon of hope!", "Filled with hope!", "Shiny!", "Absolute Hope!",
-            "The meaning of hope", "Inspiring!",
+            "The meaning of hope", "Inspiring!", "Oooh, you're a smart cookie", "I bet you sweat glitter.", "Better than bubble wrap.",
```
Comment (13xforever) [inline CompatBot/Commands/Misc.cs]: The smart cookie doesn’t really give any answer, so should be in neutral I think. Samw with glitter.

File: CompatBot/Commands/Misc.cs [hunk 6]
```diff
@@ -80,7 +84,7 @@ internal sealed class Misc: BaseCommandModuleCustom
             "Boooooooo!", "Poopy", "Smelly", "Feeling-breaker", "Annoying",
             "Boring", "Easily forgettable", "An Abomination", "A Monstrosity", "Truly horrific",
             "Filled with despair!", "Eroded by despair", "Hopeless…", "It's pretty foolish to want to rate this", "Cursed with misfortune",
-            "Nothing but terror",
+            "Nothing but terror", "Not good, at all", "Please uninstall",
```
Comment (13xforever) [inline CompatBot/Commands/Misc.cs]: “Rate this comic book”
“Please uninstall”
“Um what”

**Comment by guipleite** [comment]
I Implemented the requested changes in my last commit

File: CompatBot/Commands/Misc.cs (hunk 6 above)
Comment (guipleite) [reply CompatBot/Commands/Misc.cs]: Yeah, I think i got the wrong idea of what it should do, at first.
But now I think I fixed what you requested

**Comment by 13xforever** [comment]
Not sure you pushed the right changes, most answers just have a bunch of underscores now

**Comment by guipleite** [comment]
Crap, my bad

File: CompatBot/Commands/Misc.cs [hunk 7]
```diff
@@ -18,29 +18,32 @@ internal sealed class Misc: BaseCommandModuleCustom
...
             "Ya fo sho", "Fo shizzle mah nizzle", "Yuuuup", "Da", "Affirmative", // 5
             "Sure", "Yeah, why not", "Most likely", "Sim", "Oui",
             "Heck yeah!", "Roger that", "Aye!", "Yes without a doubt m8!", "<:cell_ok_hand:324618647857397760>",
             "Don't be an idiot. YES.", "Mhm!", "Many Yes", "Yiss", "Sir, yes, Sir!", 
             "Yah!", "Ja", "Umu!", "Make it so", "Sure thing",
             "Certainly", "Of course", "Definitely", "Indeed", "Much yes",
             "Consider it done", "Totally", "You bet", "Yup", "Yep",
-            "Certainly", "Yarp",
+            "Positive!", "Yarp", "Hmmm, yes!", "That's a yes for me", "Aye mate",
+            "Totally", "Totes my goats", "Without fail",
```
Comment (13xforever) [inline CompatBot/Commands/Misc.cs]: "totally" is a duplicate

File: CompatBot/Commands/Misc.cs [hunk 8]
```diff
@@ -18,29 +18,32 @@ internal sealed class Misc: BaseCommandModuleCustom
...
 
-            // 20
+            // 24
             "Maybe", "I don't know", "I don't care", "Who cares", "Maybe yes, maybe not",
             "Maybe not, maybe yes", "Ugh", "Probably", "Ask again later", "Error 404: answer not found",
             "Don't ask me that again", "You should think twice before asking", "You what now?", "Ask Neko", "Ask Ani",
             "Bloody hell, answering that ain't so easy", "I'm pretty sure that's illegal!", "What do *you* think?", "Only on Wednesdays", "Look in the mirror, you know the answer already",
+            "Don't know, don't care", "*visible confusion*", "Have you googled it?", "Not sure my dude",
```
Comment (13xforever) [inline CompatBot/Commands/Misc.cs]:
```suggestion
            "Don't know, don't care", "_shows signs of complete confusion_", "Have you googled it?", "Not sure my dude", 
```

File: CompatBot/Commands/Misc.cs [hunk 9]
```diff
@@ -18,29 +18,32 @@ internal sealed class Misc: BaseCommandModuleCustom
...
-            // 29
+            // 34
             "Nah mate", "Nope", "Njet", "Of course not", "Seriously no",
             "Noooooooooo", "Most likely not", "Não", "Non", "Hell no",
             "Absolutely not", "Nuh-uh!", "Nyet!", "Negatory!", "Heck no",
             "Nein!", "I think not", "I'm afraid not", "Nay", "Yesn't",
             "No way", "Certainly not", "I must say no", "Nah", "Negative",
-            "Definitely not", "No way, Jose", "Not today", "Narp",
+            "Definitely not", "No way, Jose", "Not today", "Narp", "Not in a million years", 
+            "I’m afraid I can’t let you do that Dave.", "This mission is too important for me to allow you to jeopardize it.", "Oh, I don't think so", "By *no* means",
```
Comment (13xforever) [inline CompatBot/Commands/Misc.cs]: you should use regular tick mark for consistency, not every font in every language has fancy curled ones

Comment (guipleite) [reply CompatBot/Commands/Misc.cs]: Fixed it

**Comment by 13xforever** [comment]
can you squash all of this into one single commit? if not it's ok, I's simply prefer to have this as an atomic change in the history

**Comment by 13xforever** [comment]
I think you pushed unmerged version with diffs for conflicted lines

**Comment by 13xforever** [comment]
thank you

**Pull request merged**